delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -544,6 +551,87 @@ final class Files(
     }
   }
 
+  override def migrate(
```

Comment:
I find it quite disturbing the migration code is creeping all over the rest of the code. I would much rather have it composable not inherited like this.